docs/developers/tooling/cross-chain/ccip-read-gateway.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: CCIP Read on Linea
+description: Use a CCIP Read gateway to query data on Linea from L1
+---
+
+With Alpha v3.4, the Linea team added support for Linea Name Service — the very first full 
+implementation of Ethereum Name Service (ENS) on an L2. 
+
+While this update provides many user-facing benefits, allowing Linea users to register 
+human-readable domains for considerably lower fees than on Ethereum Mainnet, it also leverages smart 
+contracts with considerable utility for developers.
+
+The [Linea ENS repository](https://github.com/Consensys/linea-ens) contains a frontend and 
+associated contracts for ENS to work on Linea, but also adds functionality that enables any L1 
```

Comment:
The topic heading is about CCIP Read, but we're introducing Linea Naming Service. Perhaps we could move the "What is CCIP Read" content upfront? We can then mention how it benefits Linea Naming Service, and any other scenarios where it is useful.

Also, we should probably avoid having version numbers scattered in the docs. Perhaps link to the release note entry if users need to know this info.